src/components/views/location/Map.tsx

```diff
@@ -51,8 +51,8 @@ const useMapWithStyle = ({ id, centerGeoUri, onError, interactive, bounds }) =>
             try {
                 const coords = parseGeoUri(centerGeoUri);
                 map.setCenter({ lon: coords.longitude, lat: coords.latitude });
-            } catch (error) {
-                logger.error('Could not set map center', centerGeoUri);
+            } catch (_error) {
```

Comment:
```suggestion
            } catch () {
```
Why not just leave the receiver blank?